UI/window-basic-settings.cpp

```diff
@@ -3384,6 +3462,90 @@ void OBSBasicSettings::ReloadAudioSources()
 	LoadAudioSources();
 }
 
+void OBSBasicSettings::MultichannelToggled(bool state)
+{
+	if (state)
+	{ //display all channels
+		ui->channelSetup->addItem("2.1");
+		ui->channelSetup->addItem("4.0 Quad");
+		ui->channelSetup->addItem("4.1");
+		ui->channelSetup->addItem("5.1");
+		ui->channelSetup->addItem("7.1");
+		blog(LOG_INFO, "Multichannel ON, bitrates > 320 kbs unlocked");
+	//display all bitrates
+		PopulateAACBitrates({ ui->simpleOutputABitrate,
+				ui->advOutTrack1Bitrate, ui->advOutTrack2Bitrate,
+				ui->advOutTrack3Bitrate, ui->advOutTrack4Bitrate,
+				ui->advOutTrack5Bitrate, ui->advOutTrack6Bitrate });
+	}	
+	else
+	{ 
```

Comment:
Style:  This curly brace and the `else` on the preceding line should be on the same block as the closing curly brace above them (lines 3480-3482).